### PR TITLE
Added solutions in R and Java

### PR DESCRIPTION
**Pull request by ericwburden**
Here's a fixed PR for this

### CODE REVIEW

**Review by drkennetz — APPROVED** [review]

File: challenges/2021-04-20-potential-palindrome/solutions/ericwburden.R [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    expect_true(can_make_palindrome(41231234))
+})
+
+test_that("'notapalindrome' does not make a palindrome", {
+    expect_false(can_make_palindrome("notapalindrome"))
+})
+
+test_that("Empty string is not a palindrome", {
+    expect_false(can_make_palindrome(""))
+})
```
Comment (parhaml) [inline challenges/2021-04-20-potential-palindrome/solutions/ericwburden.R:30]: nitpick - no newline

File: challenges/2021-04-20-potential-palindrome/solutions/ericwburden.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            xored2 = xored2 ^ b;
+        }
+        if (xored2 == 0) return true;
+
+        // If all else fails, return false
+        return false;
+    }
+}
+
+
```
Comment (parhaml) [inline challenges/2021-04-20-potential-palindrome/solutions/ericwburden.java:60]: nitpick, extra newline

**Review by parhaml — APPROVED** [review]

File: challenges/2021-04-20-potential-palindrome/solutions/ericwburden.R (hunk 1 above)
Comment (ericwburden) [reply challenges/2021-04-20-potential-palindrome/solutions/ericwburden.R:30]: I concur.

File: challenges/2021-04-20-potential-palindrome/solutions/ericwburden.java (hunk 2 above)
Comment (ericwburden) [reply challenges/2021-04-20-potential-palindrome/solutions/ericwburden.java:60]: Just making up for the .R file.

**Pull request merged**